src/Microsoft.ML.Data/Transforms/ConversionsExtensionsCatalog.cs

```diff
@@ -100,8 +100,9 @@ internal static TypeConvertingEstimator ConvertType(this TransformsCatalog.Conve
         /// <example>
         /// <format type="text/markdown">
         /// <![CDATA[
-        ///  [!code-csharp[KeyToValueMappingEstimator](~/../docs/samples/docs/samples/Microsoft.ML.Samples/Dynamic/ValueMappingStringToKeyType.cs)]
-        /// ]]></format>
+        /// [!code-csharp[ValueToKey](~/../docs/samples/docs/samples/Microsoft.ML.Samples/Dynamic/KeyToValueToKeyInputOutputPair.cs)]
```

Comment:
>KeyToValueToKeyInputOutputPair [](start = 98, length = 30)

this API isnt  for IOPair..also path looks incorrect

perhaps a case of misplaced example ?